import_tracker/rest.py

```diff
@@ -71,6 +74,22 @@ def getImports(query=None, user=None, unique=False, limit=None, offset=None, sor
     return processCursor(cursor, user)
 
 
+def moveLeafFiles(folder, user, assetstore):
+    Folder().updateFolder(folder)
+    child_items = Folder().childItems(folder)
+    child_folders = Folder().childFolders(folder, 'folder', user=user)
+
+    results = []
+    for item in child_items:
+        for (_, file) in Item().fileList(item, data=False):
+            results.append(Upload().moveFileToAssetstore(file, user, assetstore))
```

Comment:
Since this is a long running process, we probably want to call `girder.utility.progress.setResponseTimeLimit(86400)` in the loop to make sure the webserver doesn't kill the response.

Reply:
Are you suggesting to reset the response limit for each file? 

```python
    for attached_file in File().find({
        'attachedToId': folder_item['_id'],
        **unique_clause
    }):
 +      setResponseTimeLimit(86400)
        results.append(Upload().moveFileToAssetstore(attached_file, user, assetstore))

    for item in child_items:
        for file in File().find({'itemId': ObjectId(item['_id']), **unique_clause}):
 +          setResponseTimeLimit(86400)
            results.append(Upload().moveFileToAssetstore(file, user, assetstore))
```